assets/js/script.js

```diff
@@ -0,0 +1,198 @@
+$(function () {
+
+  var grid = [] //game board
+  var colorList =['#3772FF','#FED766','#FE5F55']
+  var matrix = 4 //length & breadth of board
+  var $cell = $('.cell')
+  var score = 0 // starting score
+  var time = 60 // time limit of the game
+  var checker = [] //to store player matches
+  var gameChecker =[] //to store whole board matches
+
+  function generateLevel () {
+    //creates a new 2Darray specified by 'matrix'
+    //create rows then columns, grid can be accessed with grid[row][column]
+    for (i = 0; i < matrix; i++) {
+      grid.push([])
+      for (j = 0; j < matrix; j++) {
+        grid[i].push(generateColor())
+
+        //populate cells in css
+        var pos = i +","+ j
+        $('.cell[data-position= "'+ pos +'"]').css('backgroundColor', grid[i][j])
```

Comment:
You may try : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

and need double quote mark there?